Doc/library/typing.rst

```diff
@@ -2215,6 +2215,9 @@ Corresponding to other types in :mod:`collections.abc`
 
    An alias to :class:`collections.abc.Hashable`.
 
+   .. deprecated:: 3.12
+      :class:`collections.abc.Hashable` should be directly imported from the :class:`collections.abc`.
```

Comment:
Please apply the same change below.
```suggestion
      Use :class:`collections.abc.Hashable` directly instead.
```
Also a note for future work, the reStructuredText markup for a module is `mod`. So it should be
```
:mod:`collections.abc`
```
not
```
:class:`collections.abc`
```